### PR TITLE
feat(SoundCloud): Cover as largeImageKey, non async-y strings

### PR DESCRIPTION
**Pull request by Bas950**
image

### CODE REVIEW

**Review by Slowlife01 — DISMISSED** [review]
kek

File: websites/S/SoundCloud/presence.ts [hunk 1]
```diff
@@ -154,97 +177,106 @@ presence.on("UpdateData", async () => {
...
+
+    presenceData.startTimestamp = startTimestamp;
+    presenceData.endTimestamp = endTimestamp;
+
+    if (cover) {
+      presenceData.largeImageKey = document
+        .querySelector<HTMLSpanElement>(
+          ".playbackSoundBadge__avatar.sc-media-image > div > span"
+        )
+        .style.backgroundImage.match(/"(.*)"/)[1]
```
Comment (Slowlife01) [inline websites/S/SoundCloud/presence.ts:189]: Actually, it could be null

Comment (Bas950) [reply websites/S/SoundCloud/presence.ts:183]: Actually, it cannot be null, this is inside of the if case which checks whether something is playing or not, which means the bar at the bottom of the screen is shown.

Comment (Slowlife01) [reply websites/S/SoundCloud/presence.ts:183]: *Some don't have cover*

Comment (Bas950) [reply websites/S/SoundCloud/presence.ts:183]: got an example?

Comment (Slowlife01) [reply websites/S/SoundCloud/presence.ts:183]: https://soundcloud.com/user48301249/vvkwvzuw2psc?in=user48301249/sets/heart

**Review by EncryptedDev — APPROVED** [review]

**Review by Slowlife01 — APPROVED** [review]

**Pull request merged**